.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - "0.10"
+before_script:
+  - npm install -g bower
+  - bower install
```

Comment:
this isn't really necessary b/c the tests are run against built code and we're not building the code though we really should.